# This hold basic information about the characterists of each sub-map
Header header 
float32 resolution             # The map resolution [m/cell]
uint32 num                     # num of obstacles
obs[] obsData                  # obstacles